packages/bff/src/services/selfcareService.ts

```diff
@@ -48,6 +66,48 @@ export function selfcareServiceBuilder(
         queries: { userIdForAuth: userId },
       });
     },
+
+    async getInstitutionUsers(
+      tenantId: TenantId,
+      userId: UserId | undefined,
+      query: string | undefined,
+      { authData, logger, headers }: WithLogger<BffAppContext>
+    ): Promise<bffApi.Users> {
+      logger.info(`Retrieving users for institutions ${tenantId}`);
+
+      const roles = authData.userRoles;
+      const requesterId = authData.organizationId;
+      if (!userId) {
+        throw new Error(
+          `Error while retrieving users corresponding to tenant ${tenantId}`
+        );
```

Comment:
We don't have a specific error in this situation?

Reply:
not yet, but could be more consistent using ad hoc error for this case

Reply:
the error is not more necessary, remove in [fcd8ac1](https://github.com/pagopa/interop-be-monorepo/pull/986/commits/fcd8ac146a590bf51e60f0b27f54441690688066)